esmvaltool/config-references.yml

```diff
@@ -581,153 +581,6 @@ authors:
     institute: NCAR, US
     orcid:
 
```

Comment:
How about adding a comment here, to inform users that the reference section has been replaced by bibtex files in [path-to-bibtex-files]. I would also add a note somewhere in the documentation.